test/test/testUtils.js

```diff
@@ -152,3 +142,92 @@ export const mockIntl = {
   formatHTMLMessage: () => {},
   now: () => {},
 };
+
+// React Testing Library utils
```

Comment:
what about having
- a separate file for RTL utilities
- a single wrapper including all our providers so devs don't have to think about what to use every time?

Reply:
@gidesan I agree with the separate RTL utils file.

However, I'm not sure about the single wrapper that includes all providers. If the dev writes a test for a component that they know only uses a router, then they should use the render function with the router provider. I believe it adds a very small overhead as well if all providers are included always every render.

Reply:
But the real app uses all those providers. That means the UI component tests would be a bit less "unit" and a bit more "integration", which is not bad for me.